.github/workflows/boring-open-version-bump.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           fi
       - id: check-sha-openssl
         run: |
-          SHA=$(git ls-remote https://github.com/openssl/openssl refs/heads/master | cut -f1)
+          SHA=$(git ls-remote https://github.com/openssl/openssl refs/heads/main | cut -f1)
```

Comment:
no, this is wrong

Reply:
argh, that's what I get for carelessly doing this on mobile.